mailpoet/lib/Automation/Integrations/MailPoet/Triggers/SegmentSubscribedTrigger.php

```diff
@@ -13,12 +15,18 @@
 use MailPoet\WP\Functions as WPFunctions;
 
 class SegmentSubscribedTrigger implements Trigger {
```

Comment:
You renamed the JS part, the PHP should be renamed as well. Same for the triggering hooks, local variables, etc.

Reply:
Addressed in [1106c28](https://github.com/mailpoet/mailpoet/pull/4371/commits/1106c288e7f5656ed2aa56b42058fd51c63b06e5)